infrastructure/account-data-deleter/src/lambda/base/sqsLambda.ts

```diff
@@ -29,7 +29,7 @@ export class SqsLambda extends Construct {
   ) {
     super(scope, name.toLowerCase());
 
-    const { sentryDsn, gitSha } = this.getEnvVariableValues();
+    const { sentryDsn } = this.getEnvVariableValues();
```

Comment:
We need to stop passing GitSha through via Terraform. Like ECS this is a variable that gets updated via "code deployment" and we ignore in our infrastructure.